core/src/main/scala/org/bitcoins/core/hd/HDAccount.scala

```diff
@@ -23,3 +25,39 @@ case class HDAccount(
   def toChain(chainType: HDChainType): HDChain =
     HDChain(chainType = chainType, account = this)
 }
+
+object HDAccount {
```

Comment:
I think it'd be useful to have a function that takes a `BIP32Path` as an input and returns a `HDAccount`

Reply:
Do you have anything in mind to use it for at the moment?

Reply:
I have a hww, and only want to be able to spend from account `2`, when given a `BIP32Path` I want to verify that it is the correct `HDAccount`